src/components/buttons/OutlineButtonPrimary.jsx

```diff
@@ -0,0 +1,10 @@
+export const OutlineButtonPrimary = ({ children, className, onClick }) => {
```

Comment:
suggestion: can be part of Button itself with different props
 primary, outline, disabled and so on and conditionally add class to the button based on props